_shared/project/Makefile

```diff
@@ -159,7 +137,6 @@ sure:
 # requirements/%.txt filename, for example requirements/foo.txt -> foo.
 requirements/%.txt: requirements/%.in
 	@touch -a $(subst prod.txt,dev.txt,$@)
-	@tox -qe $(subst prod,dev,$(basename $(notdir $@))) --run-command 'pip --quiet --disable-pip-version-check install pip-tools'
```

Comment:
This initial install of `pip-tools` into the virtualenvs has been moved into `tox.ini` (see below) so it's no longer needed here